dsmr_exporter/dsmr_exporter.py

```diff
@@ -181,13 +181,13 @@ def connect_elastic_output(self, host, port):
             # self.elastic_host = elasticsearch.Elasticsearch([host + ':' + str(port)], sniff_on_start=True)
             self.elastic_host = elasticsearch.Elasticsearch([host + ':' + str(port)])
         except elasticsearch.exceptions.TransportError:
-            self.logger.info("warning: elasticsearch not sniffable, continueing without sniffing")
+            self.logger.info("warning: elasticsearch not sniffable, continuing without sniffing")
```

Comment:
Note: This log doesn't make much sense if in the current code above sniffing isn't used either.